src/parseTable.ts

```diff
@@ -193,16 +201,36 @@ function getDateString(dateRaw: string | undefined): string | undefined {
  *
  * First, `/` and `\` are removed since they can't exist in file path.
  * Second, if the escaped url is a empty string or user doesn't specify an
- * url, use page id as the url.
- * @param {Nast.Page} page
- * @param {string} propId
+ * url, generate a slug from the title (if `autoSlug` is `true` in `Config`)
+ * and use it along with page id as the filename,
+ * otherwise just use the page id as is.
+ * @param {string} pageUri
+ * @param {string} customSlug
+ * @param {string} title
+ * @param {Config} config
+ * @returns {string}
+ */
+function getPageUrl(pageUri: string, customSlug: string, title: string, config: Config): string {
+  var url = getSafeUrl(customSlug)
+  if (url.length == 0) {
+    if (config.get("autoSlug")) {
```

Comment:
You may notice other places use single quotes (`'xxx'`) for strings — but this is my fault though, I haven't set up an automatic coding style checker, so it's ok. But it would be great if you could run `npx prettier src/ -w` manually to format code before commiting!

Reply:
Sounds good. If you want, I can set up husky to automatically run prettier in a separate PR (work is off and I need something to do hehe)

Reply:
Good idea! Feel free to do so.